docs/can-stache-element.md

```diff
@@ -206,6 +206,31 @@ document.body.querySelector("button#remove").addEventListener("click", () => {
 @codepen
 @highlight 14-23,only
 
+
+#### Observable class fields
```

Comment:
Can you move this under the `Defining an element's properties` heading?

Reply:
Done!